### PR TITLE
Edit typos and prettify code in util_rand_resp.rst

### PR DESCRIPTION
**Pull request by natashawatkins**
- @mmcky is there a way to cite equations in text? the equation labels in the text didn't match the automatic equation labs. I changed the references but guessed the syntax.
- I think an article stills need citation? "XXXX Ljungqvist"
- I removed the font selection because it was producing an error

### CODE REVIEW

**Comment by mmcky** [comment]
Thanks @natashawatkins.

Yes you can either use a `label` in the `math directive` or you can use `(label)` syntax immediately after a `$$`

https://jupyterbook.org/en/stable/content/math.html#numbering-equations

File: lectures/util_rand_resp.md [hunk 1]
```diff
@@ -173,19 +173,19 @@ $$
 \text{Pr}(\text{yes}|A^{'})\times \text{Pr}(A|\text{yes})
 $$ (eq:util-rand-eight-b)
 
-This measure is just the first term in $(7)$, i.e., the probability that an individual answers "yes" and is perceived to belong to A.
+This measure is just the first term in `ref:util-rand-seven-a`, i.e., the probability that an individual answers "yes" and is perceived to belong to $A$.
```
Comment (mmcky) [inline lectures/util_rand_resp.md]: This should be

```
{eq}`eq:util-rand-seven-a`
```

to use the `eq` reference role

Comment (mmcky) [reply lectures/util_rand_resp.md]: the `eq` that has been used in the actual `label` is pretty redundant so could also be removed.

**Comment by natashawatkins** [comment]
@mmcky I updated the equation references, thanks

**Comment by mmcky** [comment]
thanks @natashawatkins 

I guess preview doesn't get built as your forked repo doesn't have netlify credentials.

**Pull request merged**